Sources/StreamChat/Extensions/Task+Extensions.swift

```diff
@@ -11,3 +11,16 @@ extension Task {
         }
     }
 }
+
+extension Task {
+    static func run(_ operation: @escaping () async throws -> Void, completion: @escaping (Error?) -> Void) where Success == Void, Failure == any Error {
```

Comment:
I don't think we need this. Best to keep everything with completion handlers, and just in the state layer use a continuation wrapper.

Reply:
OK, I'll change it to completion handlers.